gammapy/makers/map.py

```diff
@@ -166,9 +167,17 @@ def make_exposure(geom, observation, use_region_center=True):
             Exposure map.
         """
         if isinstance(observation.aeff, Map):
-            return observation.aeff.interp_to_geom(
-                geom=geom,
-            )
+            aeff = observation.aeff
+            if aeff.unit.is_equivalent(u.Unit("m2 s")):
+                factor = 1.0
+            elif aeff.unit.is_equivalent(u.Unit("m2 s sday-1")):
```

Comment:
This does not guarantee that the aeff map will be properly normalized as exposure per transit as any map with a area unit will comply.
Maybe a meta information could be used instead? Or a dedicated IRF object